test/Microsoft.Sbom.Api.Tests/Filters/ManifestFolderFilterTests.cs

```diff
@@ -37,4 +52,35 @@ public void ManifestFolderFilterTest_CheckAllManifestFolder_Succeeds()
         Assert.IsFalse(filter.IsValid("c:/test/_manifest\\manifest.json"));
         configMock.VerifyAll();
     }
+
+    [TestMethod]
+    public void ManifestFolderFilterTest_CheckAllManifestFolder_Succeeds_LinuxBased()
+    {
+        // if OS is windows in the name then don't run the linux test
+        if (isWindows)
```

Comment:
This pattern keeps repeating. We should have a attribute for operating systems for test, so that we can  just say which tests are for Windows and Linux.